package com.lucas.hibernate.dao.chapter4;

import org.hibernate.HibernateException;

public interface CatDao {
	public void save() throws Exception;
}
